lib/modules/manager/setup-cfg/extract.ts

```diff
@@ -11,7 +11,7 @@ function getSectionName(str: string): string {
 }
 
 function getSectionRecord(str: string): string {
-  const [, sectionRecord] = regEx(/^([^\s]+)\s+=/).exec(str) ?? [];
+  const [, sectionRecord] = regEx(/^([^\s]+)\s*=/).exec(str) ?? [];
```

Comment:
This change allows matching the setup.cfg section record (or key) with 0 or more whitespace characters before the equal sign.